quickstart/java-redisson-jcache/README.md

```diff
@@ -10,33 +10,62 @@ products:
 ---
 # Quickstart: Use Azure Cache for Redis in Java with Redisson Redis client and JCache
 
-This sample show you how to incorporate Azure Cache for Redis into a Java app using the [Redisson](https://redisson.org/) Redis client and [JCache](https://redisson.org/glossary/jcache.html).
+In this quickstart, you incorporate Azure Cache for Redis into a Java app using the [Redisson](https://redisson.org/) Redis client and JCP standard JCache API to have access to a secure, dedicated cache that is accessible from any application within Azure.
 
 ## Prerequisites
 
 - Azure subscription - [create one for free](https://azure.microsoft.com/free/)
-- Azure Cache for Redis cache - [create one](https://docs.microsoft.com/azure/azure-cache-for-redis/quickstart-create-redis)
 - [Apache Maven](https://maven.apache.org/download.cgi)
+- [Use Microsoft Entra ID for cache authentication](https://learn.microsoft.com/azure/azure-cache-for-redis/cache-azure-active-directory-for-authentication)
+
 
 ## Set up the working environment
 
+### Option01: Authentication with Redis Key
+
 Depending on your operating system, add environment variables for your cache's **Host name** and **Primary access key**. Open a command prompt, or a terminal window, and set up the following values:
 
 ```CMD
 set REDIS_CACHE_HOSTNAME=<YOUR_HOST_NAME>.redis.cache.windows.net
 set REDIS_CACHE_KEY=<YOUR_PRIMARY_ACCESS_KEY>
```

Comment:
Recommend using the name `REDIS_ACCESS_KEY`

Reply:
Updated